content/en/docs/installation/code-signing.md

```diff
@@ -16,17 +16,33 @@ Signing keys required for verification are all available on this website, but th
 on the artifact you're trying to validate in the future. At the time of writing, all signing is done using the same underlying
 key.
 
+## Container Images / Cosign
+
+For all cert-manager versions from v1.8.0 and later, cert-manager container images are signed and verifiable using [`cosign`](https://docs.sigstore.dev/cosign/overview).
+
+The simplest way to verify signatures is to download the public key and then pass it to the cosign CLI directly:
+
+```console
+curl -sSOL https://cert-manager.io/public-keys/cert-manager-pubkey-2021-09-20.pem
+IMAGE_TAG=v1.8.0  # change as needed
```

Comment:
note: obviously this tag will fail right now because v1.8.0 hasn't been released, but I didn't want to put an alpha version here and it is clear from the text above that it's supported from v1.8.0 onwards